en/docs/develop/writing-a-custom-local-authenticator.md

```diff
@@ -1,161 +1,160 @@
 # Writing a Custom Local Authenticator
 
-The default WSO2 Identity Server authenticator is the basic authenticator. It is a local authenticator that authenticates the end users using a connected user store and the provided user name and password. When a user needs to be authenticated with the user store and authorized based on the assigned user role, you can write a custom local authenticator. Let's try out a sample scenario on this.
+A basic authenticator is available in the WSO2 Identity Server by default. A basic authenticator authenticates end users using a connected userstore and the provided username and password.
+
+With the WSO2 Identity Server, you can write your own local authenticator to define various authentication logic.
 
 ### Sample scenario
-You have an app called playground that is used for importing/exporting photos. The playground uses OpenID Connect to let the users log into it. The playground only allows the users who are assigned with the photoSharingRole user role to log in to it. (The users with other user roles are not allowed to log in to the playground application)
+You have the **playground2** app that you want users to log in with their telephone numbers instead of usernames. Once a user enters a telephone number, your authentication logic should identifiy the user and validate the user's credentials. 
```

Comment:
Is it the **playground2** app or the **pickup-dispatch** app?
@himeshsiriwardana